app/models/persistent_attachments/va_form.rb

```diff
@@ -10,7 +10,9 @@ class PersistentAttachments::VAForm < PersistentAttachment
   ).merge(
     {
       '21-0779' => { max_pages: 4, min_pages: 2 },
-      '21-509' => { max_pages: 4, min_pages: 2 }
+      '21-509' => { max_pages: 4, min_pages: 2 },
+      '21P-0518-1' => { max_pages: 2, min_pages: 2 },
+      '21P-0516-1' => { max_pages: 2, min_pages: 2 }
```

Comment:
Can you add some tests to spec/models/persistent_attachments/va_form_spec.rb?

Reply:
Eep! Good catch. Added!